packages/contracts-bedrock/src/L1/L1StandardBridge.sol

```diff
@@ -106,10 +111,18 @@ contract L1StandardBridge is StandardBridge, ISemver {
     {
         superchainConfig = _superchainConfig;
         systemConfig = _systemConfig;
-        __StandardBridge_init({
-            _messenger: _messenger,
-            _otherBridge: StandardBridge(payable(Predeploys.L2_STANDARD_BRIDGE))
-        });
+        crossDomainMessenger = _messenger;
+    }
+
+    /// @notice
```

Comment:
nit: natspec

Reply:
finding and fixing all instances of `/// @notice\n` right now.